lib/ansible/plugins/action/debug.py

```diff
@@ -61,8 +59,10 @@ def run(self, tmp=None, task_vars=None):
                             raise AnsibleUndefinedVariable
                         # If var name is same as result, try to template it
                         results = self._templar.template("{{" + results + "}}", convert_bare=True, fail_on_undefined=True)
-                except AnsibleUndefinedVariable:
+                except AnsibleUndefinedVariable as e:
                     results = "VARIABLE IS NOT DEFINED!"
```

Comment:
Grammar-wise, might as well remove the excalamation point here.  All caps is already shouting and "DEFINED!: error" is improper punctuation.